docs/Example HOWTO.md

```diff
@@ -232,68 +223,140 @@ dns2.example.com. 3600    IN      A       192.168.0.20
 ;; WHEN: Tue Jul 13 19:58:50 IST 2021
 ;; MSG SIZE  rcvd: 134
 ```
+### Using Dig to Browse NMOS Related DNS Items
 
-### Checking the `SRV` records
+We can use `dig` to do a top down browse of the NMOS services being served by BIND9. First we simply 
+check if our DNS is offering up the NMOS services by looking at all the pointers to services our DNS is 
+providing:
+
+```
+$dig _services._dns-sd._udp.example.com PTR
```

Comment:
Does the `$` here and in a few of the snippets below indicate a command prompt? Should it be the same as the snippets above which use `#` followed by a space?